tests/resources/mlflow-test-plugin/setup.py

```diff
@@ -6,12 +6,19 @@
     version="0.0.1",
     description="Test plugin for MLflow",
     packages=find_packages(),
+    # Require MLflow as a dependency of the plugin, so that plugin users can simply install
```

Comment:
I'd thought about copying this into `examples/plugins`, but decided to just reference this plugin where it currently is (under `tests/resources/mlflow-test-plugin`), as we actually run tests against this directory.